lib/generators/boring/ci/gitlab_ci/install/install_generator.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Boring
+  module Ci
+    module GitlabCi
+      class InstallGenerator < Rails::Generators::Base
+        desc 'Adds Gitlab CI to the application'
+        source_root File.expand_path("templates", __dir__)
+
+        RUBY_VERSION_FILE = ".ruby-version"
+        DEFAULT_RUBY_VERSION = '.ruby-version'
+
+        class_option :ruby_version, aliases: '-v',
+                     desc: "Tell us the ruby version which you use for the application. Default to Ruby #{DEFAULT_RUBY_VERSION}, which will cause the action to use the version specified in the #{RUBY_VERSION_FILE} file."
+
+        class_option :test_framework, aliases: '-tf', type: :string,
+                      desc: "Tell us the test framework you use for the application. Default to Minitest."
+
+        def add_gitlab_ci_configuration
+          @ruby_version = options[:ruby_version] ? options[:ruby_version] : DEFAULT_RUBY_VERSION
+          @run_test_command = test_framework
+
+          template('ci.yml', '.gitlab-ci.yml')
+          template('database.yml.tt', 'config/database.yml.ci')
+
+          if @ruby_version == DEFAULT_RUBY_VERSION && !ruby_version_file_exists?
```

Comment:
Let's remove the "!ruby_version_file_exists?" since if they are using asdf this won't work as well as if they don't have the file as required then project itself wouldn't be working in the first place so we don't have to check this separately.

Reply:
Saying that I think this whole block kind of is not needed anymore?